MosaicResidentInformationApi.Tests/V1/Controllers/MosaicControllerTests.cs

```diff
@@ -7,6 +7,7 @@
 using MosaicResidentInformationApi.V1.Boundary.Responses;
 using MosaicResidentInformationApi.V1.UseCase.Interfaces;
 using ResidentInformation = MosaicResidentInformationApi.V1.Boundary.Responses.ResidentInformation;
```

Comment:
Ah this might be what @LBHBReynolds-Carr raised in retro yesterday - it's fine to be in a situation where we need to sometimes specify full namespace for identically-named classes? (Seems OK here but just confirming.)